sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_tables_last_modified_v1/metadata.yaml

```
@@ -0,0 +1,9 @@
+friendly_name: Bigquery Query Tables Last Modified Date
+description: The last modified date of big query tables.
```

```suggestion
description: The last modified date of BigQuery tables.
```

I'd also suggest adding here that the table contains modification history rather than just one record per table since this is an incremental job.

---

This is not an incremental job.